packages/grafana-ui/src/components/Table/RowsList.tsx

```diff
@@ -218,6 +237,18 @@ export const RowsList = (props: RowsListProps) => {
         return colors;
       };
     }
+
+    // TODO: Also add option for colored background cell,
+    // and colored text cell
+    if (
+      fieldOptions !== undefined &&
+      fieldOptions.cellOptions !== undefined &&
+      (fieldOptions.cellOptions.type === TableCellDisplayMode.Auto ||
+        fieldOptions.cellOptions.type === TableCellDisplayMode.ColorBackground) &&
+      fieldOptions.cellOptions.wrapText
```

Comment:
it might be nice to break this out into a few separate consts defining what these checks are 😬

Reply:
Done!